src/tinyformat.h

```diff
@@ -898,7 +898,7 @@ class FormatListN : public FormatList
 // Special 0-arg version - MSVC says zero-sized C array in struct is nonstandard
 template<> class FormatListN<0> : public FormatList
 {
-    public: FormatListN() : FormatList(0, 0) {}
+    public: FormatListN() : FormatList(nullptr, 0) {}
```

Comment:
There may be an argument to not making style changes to `tinyformat.h`, which occasionally pulls in changes from upstream. Don't know what @laanwj's view is on this?